src/infra/k8s/manage-vms.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Google LLC
```

Comment:
@oliverchang do we want this stuff to be in another package that isn't accesible from GCE?
Also, why call it infra? I don't think this is very descriptive (unlike in oss-fuzz).

Reply:
Good point. @hogo6002 can we move this out to a top level dir?

"infra" seems OK to me, because this is describing the underlying infra.